src/ORM/Locator/LocatorAwareTrait.php

```diff
@@ -70,7 +70,7 @@ public function setTableLocator(LocatorInterface $tableLocator)
      */
     public function getTableLocator()
     {
-        if (!$this->_tableLocator) {
+        if (!isset($this->_tableLocator)) {
```

Comment:
This should be reverted or changed to `$this->_tableLocator === null`

Reply:
@ADmad, @dereuromark The problem with `$this->_tableLocator === null` and `!$this->_tableLocator` is that it results in: 
```
Undefined property: MyTest::$_tableLocator

.../Vendor/cakephp/orm/Locator/LocatorAwareTrait.php:73
.../src/TestSuite/MyTestCase.php:59
```